File: Rucksack alarm mit Sterung/rucksack_alarm_sender.js
input.onButtonEvent(Button.A, input.buttonEventClick(), function () {
    selection += -1
})
input.onButtonEvent(Button.B, input.buttonEventClick(), function () {
    selection += 1
})
radio.setGroup(1)
for (let index = 0; index < 5; index++) {
    music.playTone(262, music.beat(BeatFraction.Quarter))
    control.waitMicros(4)
}
let selection = 1
basic.forever(function () {
    if (selection == -1) {
        selection = 1
    }
    if (selection == 0) {
        radio.sendNumber(13)
        basic.setLedColor(0x00ff00)
    }
    if (selection == 1) {
        radio.sendNumber(12)
        basic.setLedColor(0xff0000)
    }
    if (selection == 2) {
        selection = 0
    }
})

File: Rucksack alarm mit Sterung/rucksack_alarm_sender.py
def on_button_a():
    global selection
    selection += -1
input.on_button_event(Button.A, input.button_event_click(), on_button_a)

def on_button_b():
    global selection
    selection += 1
input.on_button_event(Button.B, input.button_event_click(), on_button_b)

radio.set_group(1)
for index in range(5):
    music.play_tone(262, music.beat(BeatFraction.QUARTER))
    control.wait_micros(4)
selection = 1

def on_forever():
    global selection
    if selection == -1:
        selection = 1
    if selection == 0:
        radio.send_number(13)
        basic.set_led_color(0x00ff00)
    if selection == 1:
        radio.send_number(12)
        basic.set_led_color(0xff0000)
    if selection == 2:
        selection = 0
basic.forever(on_forever)
